integrated_channels/moodle/client.py

```diff
@@ -366,12 +368,12 @@ def create_content_metadata(self, serialized_data):
             # treat duplicate as successful, but only if its a single course
             # set chunk size settings to 1 if youre seeing a lot of these errors
             if error.moodle_error == 'shortnametaken' and not more_than_one_course:
-                return True
+                return 200, "success"
             elif error.moodle_error == 'courseidnumbertaken' and not more_than_one_course:
-                return True
+                return 200, "success"
             else:
                 raise error
-        return True
+        return 200, "success"
```

Comment:
should we get the response above and use the `response.text` here rather than success? maybe pass the actual return code back?